core/client/fs/pom.xml

```diff
@@ -35,6 +35,10 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Can we put this "dependency" in the top level POM?

Reply:
The root `pom.xml` has already defined the `guice` dependency, and explicit declaration is also required in submodules' `pom.xml` if the submodules want to use `guice`.